appinventor/docs/reference/components/sensors.html

```diff
@@ -612,7 +612,13 @@ <h3> Methods </h3>
       <dt> <code> number Hour(InstantInTime instant) </code> </dt>
       <dd> Returns the hour of the day(0-23) from the instant </dd>
       <dt> <code> InstantInTime MakeInstant(text from) </code> </dt>
-      <dd> Returns an instant specified by MM/dd/YYYY hh:mm:ss or MM/dd/YYYY or hh:mm. An example text input is "06/22/2015 12:18"</dd>
+      <dd> Returns an instant specified by MM/dd/YYYY hh:mm:ss or MM/dd/YYYY or hh:mm. An example text input is "06/22/2015 12:18"  </dd>
```

Comment:
It looks like the spacing on this line is modified before the closing `</dd>`. Since this isn't a significant difference, I recommend you change it back.